src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,92 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
+
 public class WorkWithFile {
+    private static final String SUPPLY_CONDITION = "supply";
+    private static final String BUY_CONDITION = "buy";
+    private static final String REGEX_CONDITION = "\\D";
+    private StringBuilder stringBuilder = new StringBuilder();
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String textFromFile = readDataFromFile(fromFileName);
+        int[] array = calculateResultForDay(textFromFile);
+        String result = toStringForNewFile(array);
+        writeToFile(toFileName, result);
+
+    }
+
+    private String readDataFromFile(String fromFileName) {
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            String value = null;
+            try {
+                value = bufferedReader.readLine();
+            } catch (IOException e) {
+                throw new RuntimeException("Can't read file " + e);
+            }
+            while (value != null) {
+                stringBuilder.append(value).append(System.lineSeparator());
+                try {
+                    value = bufferedReader.readLine();
+                } catch (IOException e) {
+                    throw new RuntimeException("Can't write file in StringBuilder " + e);
+                }
+            }
+        } catch (FileNotFoundException e) {
+            throw new RuntimeException("Can't find a file " + e);
```

Comment:
```suggestion
            throw new RuntimeException("Can't find a file " + fileName, e);
```
this would be better, same everywhere